src/Forwarder.cpp

```diff
@@ -192,12 +197,6 @@ void Forwarder::forward_epics_to_kafka() {
     auto TimeAfterIterationExecution = STEADY_CLOCK::now();
     IterationExecutionDuration = std::chrono::duration_cast<MILLISECONDS>(
```

Comment:
```suggestion
    auto IterationExecutionDuration = std::chrono::duration_cast<MILLISECONDS>(
```

Reply:
Forwarder main loop is being cleaned up as part of DM-1755